src/core/server/saved_objects/migrations/core/elastic_index.ts

```diff
@@ -28,25 +28,33 @@ export interface FullIndexInfo {
   mappings: IndexMapping;
 }
 
+/**
+ * Types that are no longer registered and need to be removed
+ */
+export const REMOVED_TYPES: string[] = [
```

Comment:
Pretty basic for now, but can be improved in https://github.com/elastic/kibana/issues/104246